attestations.py

```diff
@@ -83,7 +83,7 @@ def assert_attestations_do_not_pre_exist(
     existing_attestations_list = '\n'.join(map(str, existing_attestations))
     error_message = (
         'The following distributions already have publish attestations:'
```

Comment:
Looks like I also forgot to add a whitespace here:
```suggestion
        'The following distributions already have publish attestations: '
```